src/features/bookings/components/BookingDetailsTicketContent.tsx

```diff
@@ -49,7 +50,7 @@ export const BookingDetailsTicketContent = (props: BookingDetailsTicketContentPr
   const isDigitalAndActivationCodeEnabled =
     activationCodeFeatureEnabled && properties.hasActivationCode
 
-  const collectType = booking?.stock?.offer?.withdrawalType || ''
+  const collectType = booking?.stock?.offer?.withdrawalType || null
```

Comment:
je pense qu'`undefined` c'est une bonne valeur par défaut

```suggestion
  const collectType = booking?.stock?.offer?.withdrawalType
```

Reply:
```suggestion
  const collectType = booking?.stock?.offer?.withdrawalType || undefined
```